test/nodetool/test_status.py

```diff
@@ -201,9 +208,14 @@ def _do_test_status(nodetool, keyspace, node_list, resolve=None):
                                  response={"message": f"std::runtime_error({null_ownership_error})", "code": 500}),
                 expected_request("GET", "/storage_service/ownership", multiple=expected_request.ANY,
                                  response=ownership_response)]
+    elif table is None:
+        keyspaces_using_tablets = [keyspace] if keyspace_uses_tablets else []
+        expected_requests += [
+                expected_request("GET", "/storage_service/keyspaces", params={"replication": "tablets"}, multiple=expected_request.ANY, response=keyspaces_using_tablets),
+                expected_request("GET", f"/storage_service/ownership/{keyspace}", multiple=expected_request.ANY, response=ownership_response)]
```

Comment:
Why are you using `ANY` here? There will be just one of each request, no?

Reply:
`/storage_service/keyspaces` should be accessed just once.

`/storage_service/ownership/{keyspace}` is accessed one or zero times depending on `keyspaces_using_tablets`. I will adjust the code.

Reply:
Fixed

Reply:
Sadly, if I set the exact count, then tests run with `-nodetool=cassandra` fail. 

Cassandra's nodetool does not call `/storage_service/keyspace`, but always call `/storage_service/ownership/{keyspace}`.

So, the expectation would need to be set depending on the nodetool type used.

Reply:
But the cassandra nodetool shouldn't even execute this branch at all -- it doesn't have tablets support.

Reply:
Ah, this is the `table is None` branch.

Reply:
Maybe worth adding a branch over which nodetool variant is used here.

Reply:
> Maybe worth adding a branch over which nodetool variant is used here.

Done, thanks :+1: